src/nxdoc/nuxeo-server/upgrading-the-nuxeo-platform/upgrade-from-lts-2016-following-fast-tracks.md

```diff
@@ -399,4 +399,125 @@ Nuxeo Segment.io plugin is now aligned on `com.github.segmentio:analytics:1.0.7`
 - [Upgrade notes for 9.2](https://jira.nuxeo.com/issues/?jql=project%20in%20%28NXP%29%20AND%20resolution%20%3D%20Fixed%20AND%20fixVersion%20IN%20%28%229.2%22%20%29%20AND%20%28%22Impact%20type%22%20%3D%20%22API%20change%22%20OR%20%22Upgrade%20notes%22%20is%20not%20EMPTY%29%20ORDER%20BY%20component%20DESC%2C%20key%20DESC)
 - [Release notes for 9.2]({{page version='' space='nxdoc' page='nuxeo-server-release-notes'}})
 
+## From 9.2 to 9.3
+
+### Installation and Configuration
+
+#### New Parameters
+
+<div class="table-scroll">
+<table class="hover">
+<tbody>
+<tr>
+<th colspan="1">Parameter</th>
+<th colspan="1">Modification</th>
+<th colspan="1">Reference</th>
+</tr>
+<tr>
+<td colspan="1">`nuxeo.server.tomcat_error.show_report`</td>
+<td colspan="1">New Default Value to `false`</td>
```

Comment:
Is the parameter new or did we change the default value? Question is for the 3 parameters.